test/priv/ice_agent_test.exs

```diff
@@ -1357,6 +1371,79 @@ defmodule ExICE.Priv.ICEAgentTest do
       assert new_pair.responses_received == pair.responses_received + 1
     end
 
+    test "success response with the xor address of a local candidate with a different socket." do
+      # This test checks a specific scenario where one of the local candidates uses a socket opened on a bridge interface:
+
+      # L - local side
+      # R - remote side
+      # RC1 - remote candidate
+
+      # 1. L opens socket on interface 1 (I1), port 5000 - first local candidate (LC1)
+      # 2. L opens socket on interface 2 (I2), port 5000 - second local candidate (LC2)
+      # 3. L sends a connection check from LC1 to RC1. Given LC1 operates via I1, which is a bridge interface, its source address is rewritten to I2
+      # 4. R perceives the request from L as originating from I2, port 5000, and responds successfully to I2, port 5000
+      # 5. This response arrives to the I1 port 5000. L notices that R recognized it as coming from I2, port 5000
+      # 6. L chooses to use LC1 and RC1 as the discovered pair because we know that I1 is a bridge interface.
+
+      # Note: If we were to use LC2 and RC1 as the discovered pair
+      # we would have different sockets between the succeeded and discovered pairs, which would cause a runtime error.
```

Comment:
Just describe what the test aims to check. We have detailed description in the main logic